llvm/include/llvm/MC/MCRegisterInfo.h

```diff
@@ -382,6 +385,10 @@ class MCRegisterInfo {
     return RegStrings + get(RegNo).Name;
   }
 
+  bool isConstant(MCRegister RegNo) const {
```

Comment:
Can you add a docstring to this function?

Reply:
Oh yeah, absolutely.